README.md

```diff
@@ -156,11 +156,11 @@ minio during development.
 
 ### Prince XML Lambda
 
-EASi runs [Prince XML](https://www.princexml.com/) as a Lambda function to
+MINT runs [Prince XML](https://www.princexml.com/) as a Lambda function to
 convert HTML to PDF.
 
 See the
-[easi-infra-modules](https://github.com/CMSgov/easi-infra-modules/blob/master/lambda/prince/README.md)
+[mint-infra-modules](https://github.com/CMSgov/mint-infra-modules/blob/master/lambda/prince/README.md)
```

Comment:
I think this can also go, both because it's Prince related and because all our infra code is in a single repo (not in one called `mint-infra-modules`)